### ISSUE TITLE
payload for A9LH ! please ! i love this tool ! :v 

### ISSUE THREAD

**Issue by NgDTien** [roxas75/rxTools#439]
the title say it all ::::

**Comment by Zetchzie** [comment]
I dont think there messing with a9lh yet.

**Comment by joedude1635** [comment]
OMG same. I like the speed of A9LH emuNAND, but don't really like reiNAND.

**Comment by Desterly** [comment]
This is already another item that's being discussed:
https://github.com/roxas75/rxTools/issues/425

I don't think we need two issues for the same topic.

**Comment by mariogamer2** [comment]
@NgDTien @AlbertoSONIC  please close

**Comment by Zetchzie** [comment]
whats the speed difference other than it starting up faster?

**Comment by urherenow** [comment]
It's not just a speed difference (which is neglegable) it's 100% boot rate.

**Comment by Zetchzie** [comment]
Ah i see. id like to know how to use a9lh but from what i read you cant get it to work unless you dump somthing on an old firmware. and im afraid its dangerous.

**Comment by urherenow** [comment]
Yes, you need your OTP. It's technically EXTREMELY dangerous, but I've seen countless noobs do it. Only bricks I've heard of were from user error, like trying to install a9lh that was compiled with someone else's OTP.bin.

**Comment by mariogamer2** [comment]
@urherenow Now,the OTP dump is more safe,and it's not extremely dangerous.

After getting your OTP @Zetchzie , you have to build this: https://github.com/delebile/arm9loaderhax (delete the folder arm9loaderhax/payload_installer/brahma2/data/ as it is not needed,and place your otp.bin, and these file: https://github.com/capito27/arm9loaderhax/tree/master/data_input (NOT THE OTP,OR YOU WILL BRICK) in arm9loaderhax/data_input)

**Comment by mariogamer2** [comment]
@AlbertoSONIC If you are working on a9hl,do not forgot screen init.

**Comment by Zetchzie** [comment]
How you supposed to get it on n3ds you cant downgrade to 2x on it.

**Comment by mariogamer2** [comment]
Go to this or this file:

https://github.com/Plailect/OTP/blob/master/New_3DS_Cubic.md  (With Cubic Ninja)
https://github.com/Plailect/OTP/blob/master/New_3DS_Spider.md (Without Cubic Ninja)

**Comment by AlbertoSONIC** [comment]
@mariogamer2 Don't use delebile arm9loaderhax, use darksamus' form as it includes screen init ;) https://github.com/dark-samus/arm9loaderhax

**Comment by mariogamer2** [comment]
https://github.com/delebile/arm9loaderhax/pull/9

**Comment by AlbertoSONIC** [comment]
Ah now delebile repo has screen init? Well then use it ;)

**Comment by Desterly** [comment]
Those using the same firmware on both n3ds and o3ds?  For compiling the payload I'm referring to

**Comment by mariogamer2** [comment]
Yes,see capito27's fork,in data_input folder.

**Comment by Zetchzie** [comment]
nope lol im not messing with a9lh im just fine with the method i got. Im lucky i didnt brick my n3ds the first time i tried downgrading it. I did brick it very minerly i had to upgrade and downgrade again. as long as the other methods stay supported i will steer clear of a9lh.

**Comment by urherenow** [comment]
@Zetchzie yes you can (downgrade n3ds to 2.1). Read the OTP guide.

**Comment by mariogamer2** [comment]
Also,just for remember:

_**Never,Never,Never use a build by another people who was build with another OTP than yours,because the file arm9loaderhax.3dsx is crypted with YOUR OTP.**_

**Comment by Sev501** [comment]
~~How do I compile the arm9loaderhax ver of rxtools? IIRC something to type instead of make release but make arm9(or sth).~~

EDIT: Will just wait and see how this progresses, until then I'm gonna use the last version that I compiled. Thank you for making this project happen.

**Comment by Desterly** [comment]
You need to checkout the 'armloaderhax' branch then compile as normal.

**Comment by mariogamer2** [comment]
Just to say like this....
I know it's not the place but want to say it...

@ Pasta Team and other dev, continue your great work in the open path.

Also do not work to hard, I know how it's difficult to support all these pressure....

Good luck...

mario

**Comment by dogtopus** [comment]
<strike>@mariogamer2 happy April Fool's day! :D</strike> 
Very clever, man, very clever...

**Comment by mariogamer2** [comment]
? lol 
I'm serious....Cake is really "shutdowned" (the source)

**Comment by mariogamer2** [comment]
Lol happy April month XD

But I lets the message,just like this XD
